frontend/src/composables/WorkSpace/useDomainData.js

```diff
@@ -98,8 +98,19 @@ export default function useDomainData() {
 	};
 
 	const addTagtoDomain = async (tag) => {
+		try {
 			await API.post(`/domain/${choosenNodeID.value}/tag/${tag.id}`);
-			refreshDomainTag(choosenNodeID.value);
+			await refreshDomainTag(choosenNodeID.value);
+			quasar.notify({
+				type: "positive",
+				message: "Tag has been successfully added",
+			});
+		} catch (error) {
+			quasar.notify({
+				type: "negative",
+				message: "Sorry, tag has not been added",
```

Comment:
Maybe add an explanation for the user : like "Error while adding tag : <reason>"
the reason being the response from the API (parsed of course)